LICENSE

```diff
@@ -1,9 +0,0 @@
-Copyright (c) 2013, Cognisant Research
```

Comment:
Why was the License file deleted?

Reply:
That's a very good question... Re-added it, not sure what happened there.